crowddata/mixins/adpem/adpem.py

```diff
@@ -15,6 +16,9 @@
 from crowddata.mixins.adpem.menu.closable_embed import ClosableEmbedMenu
 from crowddata.mixins.adpem.view.show_stats import ShowStatsView, ShowStatsViewProps
 
+# This is the timestamp of the most recent AdPEM reset
+LAST_RESET = 1640246400
```

Comment:
can we make this human readable 2021-12-19 or whatever and then convert it inside the code? then it's really easy to tell if it's up-to-date or not